client/my-sites/plan-price/index.jsx

```diff
@@ -75,7 +75,10 @@ export class PlanPrice extends Component {
 
 		const renderPrice = priceObj => {
 			const fraction = priceObj.raw - priceObj.price.integer > 0 && priceObj.price.fraction;
-			return `${ priceObj.price.integer }${ fraction }`;
+			if ( fraction ) {
+				return `${ priceObj.price.integer }${ fraction }`;
```

Comment:
An alternative way to shorten it could be just output `fraction || ''` but this works too. :-)